src/npr/rules_dorado/06_align.smk

```diff
@@ -38,11 +35,19 @@ rule align:
         genome =  config['genome'].get(org, None),
         json = target_json,
         html = target_html,
+        do_align = config['info_dict']['do_align']
     threads:
-        10
+        10 # same number of threads will be applied to dorado-align and samtools
     shell:
         """
-        {dorado} aligner {params.genome} {input.bam_file} | samtools sort -@ {threads} -m 20G - > {output.file} 2>> {log}
+        echo "do_align: {params.do_align}" 2>> {log}
+        if [[ "{params.do_align}" == "do_align" ]]; then
+            {dorado} aligner -t {threads} {params.genome} {input.fq_file} | samtools sort -@ {threads} -m 20G - > {output.file} 2>> {log}
```

Comment:
Does inputting from fastq retain modification information in the aligned bam file ?

Reply:
no, during align that information is deleted, that is why if we have a BAM already aligned from the sequencer server, we don't align it again